rest/src/main/java/discord4j/rest/util/InteractionResponseType.java

```diff
@@ -58,6 +58,7 @@ public enum InteractionResponseType {
     /**
      * Response to a supported interaction with a notification that this interaction is only available to premium guilds or users.
      */
+    @Deprecated
```

Comment:
```suggestion
    @Deprecated // in favor of using {@link discord4j.core.object.component.Button#premium(Snowflake)}. This will continue to function but may be eventually unsupported
```

Reply:
i mean in the docs not in the code